### DISCUSSION TITLE
React-Native Implementation of Wallet-Connect v2

### DISCUSSION THREAD

**Discussion by daniyalayyaz** [WalletConnect/walletconnect-monorepo#1104] [Developer Support]
Hi Everyone.

I was wondering what the plans for the React-Native implementation of Wallet-Connect, v2 were.

I understand the entire v2 is still in open-beta, but I have been trying to implement react-native and am running into a lot of issues.
Is it still too early to be trying to implement it?

Are there any incoming updates that will make the process easier or update the react-native documents.

Any information would help a lot. Thank you for your excellent work!

**Comment by chadyj** [comment]
v2 is in beta, but several wallets are using it in production already. Look out for an announcement soon about future plans and releases.

We appreciate your feedback and have plans to add a react native example app. But in the meantime, you can look at our sample web examples https://github.com/WalletConnect/web-examples

**Comment by linxianxi** [comment]
same here! Documentation for the react-native v2 is minimal and almost non-existent

**Comment by chadyj** [comment]
Update: we created an issue to track progress https://github.com/WalletConnect/walletconnect-monorepo/issues/1118
